support-frontend/assets/helpers/render.js

```diff
@@ -20,6 +20,8 @@ const getElementOrBody = (id: ?string): HTMLElement => {
 };
 
 const renderError = (e: Error, id: ?string) => {
+  fetch(window.guardian.settings.metricUrl, {'mode': 'no-cors'}); // ignore result, fire and forget
```

Comment:
this uses window.guardian directly, as it is run when something has gone badly wrong, we don't want to rely on too much working at that point.  The other option is to put the getSettings outside the try block, but that is something that could easily break when adding settings (I think there was some issue recently with pricing missing)